### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by devkanro** [devkanro/intellij-protobuf-plugin#10] [General]
### Added
- Add quick document with commonmark
- Add 'json_name' option support
- Add 'allow_alias' option support

### Fixed
- 'default' option value type annotation

<hr /><em>This discussion was created from the release <a href='https://github.com/devkanro/intellij-protobuf-plugin/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by null-enjoyer** [comment]


**Reply by devkanro** [reply]
If you can provide a .NET sample project, and descript how you use protos, maybe we can try on it.

**Reply by devkanro** [reply]
Are there some issues when you use the plugin in Rider?
